cookies/templates/collection.html

```diff
@@ -127,4 +130,8 @@ <h4 class="panel-title">
 </div>
 {% endif %}
 
+<div class="text-center">
+    <a class="btn btn-success" href="{% url "create-collection" %}">Create a sub collection</a>
```

Comment:
Good idea to re-use the create-collection view. The one piece of information that we won't have in that view is the collection to which this sub-collection should be added. Perhaps you can pass that as a GET parameter, and then incorporate some additional logic in the create-collection view to handle cases when that parameter is present?